charts/fybrik/templates/_helpers.tpl

```diff
@@ -99,3 +99,15 @@ certmanager.k8s.io/v1alpha1
 cert-manager.io/v1alpha2
 {{- end -}}
 {{- end -}}
+
+{{/*
+Get blueprints namespace
+*/}}
+{{- define "fybrik.getBlueprintNamespace" -}}
```

Comment:
FYI there is a built-in `default` function to achieve the same

Reply:
leaving as is for now